### PR TITLE
docs: update getting started guide with initial feedback

### PR DESCRIPTION
**Pull request by avasisht23**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
The focus of this PR is to update file extensions, remove unused imports, and refine content related to modular accounts and authentication.

### Detailed summary
- Updated file extensions in `package.json` for `.tsx` files
- Removed unused file extensions in `globals.css` import
- Refactored references to `Modular Smart Contract Account` to `Modular Account`
- Updated content related to `Magic Link Authentication` in `terms.md`

> The following files were skipped due to too many changes: `site/getting-started/overview.md`, `site/getting-started/send-uos.md`, `site/getting-started/log-in-users.md`, `site/getting-started/setup-app.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by rthomare — APPROVED** [review]

**Review by SahilAujla — DISMISSED** [review]
Looks great @avasisht23!

Only thing is, we need to add the file extensions back in imports, otherwise the linting fails. 

image

Once we've added them back, we can merge, so approving now.

**Comment by avasisht23** [comment]
Yeah we should ignore the linter for the `site/snippets` folder.

Leeds was copying those files directly from docs into his code and they were failing. Clear signal of confusion when following this doc to a tee.

File: site/getting-started/introduction.md
Comment (moldy530) [inline site/getting-started/introduction.md:23]:
```suggestion
This guide will show you how to create Embedded Accounts using [Account Kit](https://accountkit.alchemy.com/resources/terms.html#account-kit), the Alchemy [Signer](https://accountkit.alchemy.com/resources/terms.html#signer), and the Alchemy [Modular Account](https://accountkit.alchemy.com/resources/terms.html#modular-account).
```

Comment (moldy530) [reply site/getting-started/introduction.md:23]: we're just calling it MA everywhere now so we should do a pass and make sure that holds in our docs as well

File: site/getting-started/introduction.md
Comment (moldy530) [inline site/getting-started/introduction.md:29]: I think in the original mocks we had this at the very top no?

File: site/getting-started/setup-app.md
Comment (moldy530) [inline site/getting-started/setup-app.md:53]: what's `magic auth` here?

should it be `magic link auth`?

**Review by moldy530 — APPROVED** [review]
couple comments but LGTM overall

**Review by moldy530 — APPROVED** [review]

**Pull request merged**
